x-pack/plugins/gis/public/components/map/mb/view.js

```diff
@@ -61,7 +62,12 @@ export class MBMapContainer extends React.Component {
   }
 
   async _initializeMap() {
-    this._mbMap = await createMbMapInstance(this.refs.mapContainer, this.props.goto);
+
+    try {
```

Comment:
you don't need the try/catch block if you are just going to immediately re-throw the error without doing anything with it